orchagent/neighorch.h

```diff
@@ -23,14 +23,15 @@ typedef map<NeighborEntry, MacAddress> NeighborTable;
 class NeighOrch : public Orch
 {
 public:
-
     NeighOrch(DBConnector *db, string tableName, PortsOrch *portsOrch, RouteOrch *routeOrch) :
         Orch(db, tableName), m_portsOrch(portsOrch), m_routeOrch(routeOrch) {};
+    const char* const delimiter         = ";";
+    const char* const neigh_field_name  = "neigh";
 private:
     PortsOrch *m_portsOrch;
     RouteOrch *m_routeOrch;
 
-    void doTask();
+    virtual void doTask(_in_ Consumer& consumer_info);
```

Comment:
why it is virtual? and i don't think it's necessary to add _in_ here. let's make the code unified everywhere. We are not adding this here. The reason SAI header use _in_ and _out_ is because that is an interface API definition file. These are pure codes and coder shall be able to get the idea easily.